docs/loading-aia-and-hmi.rst

```diff
@@ -0,0 +1,58 @@
+.. _loading_aia_and_hmi_files:
+
+========================================================
+Loading and Overplotting AIA and HMI files as SunPy Maps
+========================================================
+
+Loading an AIA file
+-------------------
+To load an AIA file as a SunPy Map with the ``glue-solar`` plugin of ``glue`` is
+very straightforward. Upon firing up ``glue`` while the environment is with
+``glue-solar`` installed, simply import data with the third icon from the left on the top
+navigation bar of ``glue``, choose the file one would like to view, then before clicking
+the "Open" button on the lower right corner of the loader select from the dropdown menu
+at the bottom the "SunPy Map" format. Otherwise the AIA file would be loaded automatically
```

Comment:
Outside the scope of this PR but is there anyway to automate this aka if I open a AIA file glue/glue-solar figures it out and loads the correct loader?

Reply:
I think in the long term that is possible, but will need to work out the details in time though. I can say make the SunPy maps importer do this for us as long as the data is solar ones.